server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/Serializer.scala

```diff
@@ -230,8 +230,15 @@ class Serializer @Inject() (mailboxIdFactory: MailboxId.Factory) {
       })
     }
 
-  private def mailboxWritesWithFilteredProperties(properties: Option[Properties], capabilities: Set[CapabilityIdentifier]): Writes[Mailbox] = {
-    mailboxWrites(Mailbox.propertiesFiltered(properties, capabilities))
+  private def mailboxWritesWithFilteredProperties(properties: Option[Properties], capabilities: immutable.Set[CapabilityIdentifier]): Writes[Mailbox] = {
```

Comment:
the `properties` parameter is not used anymore. Is it a  regression ?

Reply:
it seems that you ignored the change from JAMES-3098

Reply:
You should launch the `MailboxGetMethodContract` test

Reply:
oh, i will recheck it.